openflow13/nx_action.go

```diff
@@ -1471,3 +1472,451 @@ func NewNXActionController(controllerID uint16) *NXActionController {
 	a.Length = a.NXActionHeader.Len() + 6
 	return a
 }
+
+const (
+	NXAC2PT_MAX_LEN       = iota /* ovs_be16 max bytes to send (default all). */
+	NXAC2PT_CONTROLLER_ID        /* ovs_be16 dest controller ID (default 0). */
+	NXAC2PT_REASON               /* uint8_t reason (OFPR_*), default 0. */
+	NXAC2PT_USERDATA             /* Data to copy into NXPINT_USERDATA. */
+	NXAC2PT_PAUSE                /* Flag to pause pipeline to resume later. */
+	NXAC2PT_METER_ID             /* ovs_b32 meter (default NX_CTLR_NO_METER). */
+
+	NX_CTLR_NO_METER = 0
+)
+
+type NXActionController2PropMaxLen struct {
+	*PropHeader /* Type: NXAC2PT_MAX_LEN */
+	MaxLen      uint16
+	pad         [2]uint8
+}
+
+func (a *NXActionController2PropMaxLen) Len() uint16 {
+	return a.PropHeader.Len() + 2 + 2
+}
+
+func (a *NXActionController2PropMaxLen) MarshalBinary() (data []byte, err error) {
+	data = make([]byte, a.Len())
+	var b []byte
+	n := 0
+
+	a.Length = a.PropHeader.Len() + 2
```

Comment:
Why not a.Length  =a.PropHeader.Len() + 4? I don't think ignore the length of padding is a good idea.